functions/test/testing.js

```diff
@@ -0,0 +1 @@
+var myFunctions = require('../index');
```

Comment:
What's the purpose of this file? Right now it has no tests :)
Remove it for now (you can add it if you add tests)